doc/api_conf.rst

```diff
@@ -48,6 +48,11 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
 
     Debug messages output level, in the range 0 to 10. Default is 2.
 
+  .. attribute:: deltarpm_percentage
+
+    Integer option. When the relative size of delta vs pkg is larger than this, delta is not used. Default value is 75 (%).
+    Use `0' to turn off delta rpm processing. Local repositories (with file:// baseurl) have delta rpms turned off by default.
```

Comment:
I'd better rephrase it as 'Local repositories ... have delta rpms always turned off.' because 'by default' pretends it could be changed (which I think is not possible).